src/components/AudioInfo.vue

```diff
@@ -695,6 +898,18 @@ export default defineComponent({
         volumeScale: volumeScaleSlider.state.currentValue.value,
         prePhonemeLength: prePhonemeLengthSlider.state.currentValue.value,
         postPhonemeLength: postPhonemeLengthSlider.state.currentValue.value,
+        morphingInfo:
+          morphingTargetStyleInfo.value &&
+          morphingTargetCharacterInfo.value &&
+          morphingRateSlider.state.currentValue.value
```

Comment:
numberなのでこうかなと。

あと本当はここの上と同じようにthrowしたほうが良いんですけど、if文ちょっと複雑になっちゃいますね･･･。
FIXMEコメント書いとくのどうでしょう。
```suggestion
          morphingRateSlider.state.currentValue.value != undefined // FIXME: ifでチェックしてthrowする
```